src/store/store.jsx

```diff
@@ -0,0 +1,9 @@
+import { createStore, applyMiddleware } from "redux";
+import {thunk} from "redux-thunk";
+import reducers from "./reducers";
+
+console.log(thunk)
+const enahncer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__;
```

Comment:
I'm not sure, but I think you need to add check here to check if there's a dev tools and there's a window before adding it as Middleware

Also u might check if this is dev or production, so no need to pass in case of production build